03_SageMakerPipelinesAndDeploy/pipeline.R

```diff
@@ -0,0 +1,161 @@
+library(reticulate)
+
+local_path <- dirname(rstudioapi::getSourceEditorContext()$path)
+
+get_pipeline <- function(input_data_uri){
+  
+  s3_raw_data <- sagemaker$workflow$parameters$ParameterString(
+    name="InputData",
+    default_value=input_data_uri
+  )
+  
+  sagemaker <- import('sagemaker')
+  boto3 <- import('boto3')
+  
+  session <- sagemaker$Session()
+  bucket <- session$default_bucket()
+  
+  role_arn <- sagemaker$get_execution_role()
+  account_id <- boto3$client("sts")$get_caller_identity()$"Account"
```

Comment:
I would typically prefer we use the sagemaker sdk with:
```
session = sagemaker$Session()
account_id <- session$account_id()
```

Okay to leave this if you feel its better to show boto3 interaction as well though

Reply:
I think I prefer your way here, it is simpler. Using boto3 has some merits but maybe here is unnecessarily confusing